android/KMEA/app/src/main/java/com/keyman/engine/KMTextView.java

```diff
@@ -167,7 +167,7 @@ protected void onTextChanged(CharSequence text, int start, int lengthBefore, int
   protected void onSelectionChanged(int selStart, int selEnd) {
     super.onSelectionChanged(selStart, selEnd);
     if (activeView != null && activeView.equals(this)) {
-      if (KMManager.updateSelectionRange(KMManager.KeyboardType.KEYBOARD_TYPE_INAPP, selStart, selEnd)) {
+      if (KMManager.updateSelectionRange(KMManager.KeyboardType.KEYBOARD_TYPE_INAPP, selStart, selEnd) && !KMManager.InAppKeyboardShouldIgnoreSelectionChange) {
```

Comment:
This looks like new behaviour? Can you explain why this has changed? (Comparing to stable-16.0)

Reply:
ah, looks like this wasn't needed. Reverted